api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/TelephonyDto.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.payment.api.contract;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+@Builder(builderMethodName = "telephonyDtoWith")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class TelephonyDto {
```

Comment:
I suggest changing DTO name to telephonyCardPaymentsResponse to represent the API

Reply:
Fixed.